setup.py

```diff
@@ -27,20 +27,10 @@
     python_requires=">=3.6.*",
     install_requires=[
         'clickhouse-driver~=0.1.2',
-        'apache-airflow~=1.10.6',
+        'apache-airflow>=2.0.0',
     ],
     license=about['__license__'],
     zip_safe=False,
-    entry_points={
```

Comment:
Could you please share a link to the documentation that states this is not necessary anymore for Airflow v2?

Reply:
I didn't find any docs that was stating that but this was the error I was getting with the old plugin definition.

```
/usr/local/lib/python3.6/site-packages/airflow_clickhouse_plugin/__init__.py:7: FutureWarning: Registering operators or sensors in plugins is deprecated -- these should be treated like 'plain' python modules, and imported normally in DAGs.

Airflow 2.0 has removed the ability to register these types in plugins. See <http://airflow.apache.org/docs/stable/howto/custom-operator.html>.
  class ClickHouseOperatorPlugin(AirflowPlugin):
/usr/local/lib/python3.6/site-packages/airflow_clickhouse_plugin/__init__.py:17: FutureWarning: Registering operators or sensors in plugins is deprecated -- these should be treated like 'plain' python modules, and imported normally in DAGs.

Airflow 2.0 has removed the ability to register these types in plugins. See <http://airflow.apache.org/docs/stable/howto/custom-operator.html>.
```
Removing the Classes, using directly the Operator and the Hook resolved the issue. If you go here you can see that the plugin is not anymore defined creating a custom Operator: http://airflow.apache.org/docs/stable/howto/custom-operator.html

Reply:
In this link the change is more clear https://github.com/apache/airflow/blob/2.0.0/UPDATING.md#custom-executors-is-loaded-using-full-import-path